project.owasp.yaml

```
@@ -0,0 +1,279 @@
+audience:
+  - builder
+  - defender
+community:
+  - name: proj-dependency-check
+    platform: slack
+    url: https://owasp.slack.com/archives/C2FTJJ26N
+    description: channel for the https://www.owasp.org/index.php/OWASP_Dependency_Check project - For general questions with the tool, check out the mailing
+      list https://groups.google.com/forum/#!forum/dependency-check. To report issues, please submit a GitHub issue.
+  - name: dependency-check
+    platform: slack
+    url: https://owasp.slack.com/archives/C082E5KB4HJ
+leaders:
+  - name: Jeremy Long
+    email: jeremy.long@owasp.org
+    github: jeremylong
+level: 4
+license:
+  - Apache-2.0
+  - EUPL-1.2
+  - GPL-3.0
+  - MIT
+name: OWASP Dependency-Check
+pitch: Dependency-Check is a Software Composition Analysis (SCA) tool suite that identifies project dependencies and checks if there are any known, publicly
+  disclosed, vulnerabilities.
+repositories:
+  - name: www-project-dependency-check
+    url: https://github.com/OWASP/www-project-dependency-check
+    description: OWASP Foundation Web Repository
+  - name: sbt-dependency-check
+    url: https://github.com/albuch/sbt-dependency-check
+    description: 'SBT Plugin for OWASP DependencyCheck. Monitor your dependencies and report if there are any publicly known vulnerabilities (e.g. CVEs).
+      :rainbow:'
+  - name: azuredevops
```

```suggestion
  - name: dependency-check-azure-devops-extension
```

Maybe can make these names more consistent/descriptive, if they don't need to match the repo name.